pkg/xen/Dockerfile

```diff
@@ -8,19 +8,34 @@ ENV BUILD_PKGS_arm64 u-boot-tools
 RUN eve-alpine-deploy.sh
 
 ENV XEN_UBOOT_ADDR 0x81000000
-ENV XEN_VERSION 4.15.0
+ENV XEN_VERSION 4.19.0-rc4
 ENV XEN_SOURCE=https://downloads.xenproject.org/release/xen/${XEN_VERSION}/xen-${XEN_VERSION}.tar.gz
 
+
+
+
+#ADD https://github.com/ninja-build/ninja.git#release /ninja
+#WORKDIR /ninja
+#RUN ./configure.py --bootstrap
+
+
 # Download and verify xen
 #TODO: verify Xen
 # hadolint ignore=DL3020
 ADD ${XEN_SOURCE} /xen.tar.gz
 RUN [ -f "$(basename ${XEN_SOURCE})" ] || tar --absolute-names -xz < /xen.tar.gz && mv "/xen-${XEN_VERSION}" /xen
 
+ADD https://github.com/ninja-build/ninja.git#release /ninja
```

Comment:
Is there any specific requirement to use this ninja release? Alpine's ninja wouldn't work? For instance, wwan package uses alpine's ninja package: https://github.com/lf-edge/eve/blob/master/pkg/wwan/Dockerfile#L7